README.md

```diff
@@ -51,30 +51,34 @@ rules:
     valueFactor: 0.001
     labels: {}
     help: "Cassandra metric $1 $2"
+    matchBeanValue: true
```

Comment:
We don't need two settings, they amount to the same thing.

Reply:
Makes sense, thanks!